server/server.go

```diff
@@ -46,11 +46,12 @@ type Server struct {
 // New constructs a new HTTP proxy server using the given options
 func New(opts *Opts) *Server {
 	p, _ := proxy.New(&proxy.Opts{
-		IdleTimeout:        opts.IdleTimeout,
-		Dial:               opts.Dial,
-		Filter:             opts.Filter,
-		BufferSource:       buffers.Pool(),
-		OKWaitsForUpstream: true,
+		IdleTimeout:         opts.IdleTimeout,
+		Dial:                opts.Dial,
+		Filter:              opts.Filter,
+		BufferSource:        buffers.Pool(),
+		OKWaitsForUpstream:  true,
+		OKSendsServerTiming: true,
```

Comment:
@myleshorton This needs to be enabled for https://github.com/getlantern/proxy/pull/28 to take effect, so I'll go ahead to merge this.